src/workloads/query/query_stats/QueryStats.tmpl

```diff
@@ -0,0 +1,276 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This is a template of predefined workload groups for QueryStats workloads.
+
+# Workload "constants."
+ConstantDatabase: &Database {^Parameter: {Name: Database, Default: test}}
```

Comment:
Why describe this as a parameter, if you're never actually using it as a set-able parameter (unless I'm missing where you're setting it)?
I'm not convinced it's worth putting these "constants" in the template file. I understand the intent of trying to factor out as many commonalities as possible, but I think each one makes it slightly more confusing to a reader trying to understand a workload file (by adding one more level of indirection), and each of those small bits is adding up to an experience where it's taking me a while to make sense of all of these workloads since the bits are so de-centralized. I wonder if there's a balance to strike with a bit less abstraction.